### PR TITLE
Sponsors: change level datatype from str to int

### LINKED ISSUE #4201: Sponsors: datatype of level should be int instead of string
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-orga-server

Sponsors: datatype of level should be int instead of string.

### PR DESCRIPTION
**Pull request by shubham-padia**
Fixes #4201

### CODE REVIEW

File: migrations/versions/dbb9670cd902_.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'dbb9670cd902'
+down_revision = 'f63b9b9b645b'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('sponsors', 'level',
```
Comment (niranjan94) [inline migrations/versions/dbb9670cd902_.py]: @shubham-padia how is the old data handled here ?

Comment (shubham-padia) [inline migrations/versions/dbb9670cd902_.py]: Sorry, missed that :grimacing: , should I delete the sponsors with invalid level or should they be changed to NULL ?

**Comment by niranjan94** [comment]
Make it null

**Comment by shubham-padia** [comment]
@niranjan94 Changes done. Please review :)

**Pull request merged**